raiden/tests/integration/cli/conftest.py

```diff
@@ -10,41 +10,38 @@
 from raiden.tests.utils.smoketest import setup_raiden, setup_testchain
 
 
-@pytest.fixture(scope="session")
-def testchain_provider(blockchain_type, port_generator):
-    eth_client = EthClient(blockchain_type)
-    chain_manager = setup_testchain(
-        eth_client=eth_client, print_step=lambda x: None, free_port_generator=port_generator
-    )
-
-    with chain_manager as testchain:
-        yield testchain
-
-
 @pytest.fixture(scope="module")
 def cli_tests_contracts_version():
     return RED_EYES_CONTRACT_VERSION
 
 
 @pytest.fixture(scope="module")
-def raiden_testchain(testchain_provider, cli_tests_contracts_version):
+def raiden_testchain(blockchain_type, port_generator, cli_tests_contracts_version):
```

Comment:
The fixture `testchain_provider` was only used here, so I removed it and added executed `setup_testchain` directly

Reply:
Not quite. `testchain_provider` was `session` scoped, this is `module`. That means we restart the eth node for every module now.

It's probably not a huge deal, but arguably we want to move to restarting less stuff for every test, not more.

Reply:
Humm, good point, I missed that.